modules/core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Bundle-Name: org.nuxeo.ecm.core.storage.sql.test
```

Comment:
this is making eclipse confuse this bundle and the nuxeo-core-storage-sql-test bundle: will rename it (see https://jira.nuxeo.com/browse/NXP-30098)